jscomp/others/js_null_undefined.ml

```diff
@@ -31,10 +31,15 @@ external test : 'a t -> bool =  "#is_nil_undef"
 external null : 'a t = "null" [@@bs.val]
 external undefined : 'a t = "undefined" [@@bs.val]
 
-let bind x f =
-  match to_opt x with
-  | None -> (Obj.magic (x: 'a t): 'b t)
-  | Some x -> return (f x [@bs])
+let map f x = match to_opt x with
+| None -> (Obj.magic (x: 'a t): 'b t)
```

Comment:
can we avoid dependency on ocaml-stdlib, i.e, obj module?
how about adding one function `Js.coerceUnsafe`

Reply:
Sure, let me see if I can get rid of that dependency.